### PR TITLE
Ie csv export

### LINKED ISSUE #13: export csv in ie
image

### PR DESCRIPTION
**Pull request by kahneraja**
fixes #13

### CODE REVIEW

File: src/components/MainView.js [hunk 1]
```diff
@@ -33,19 +33,20 @@ export default class MainView extends React.Component {
...
-        evt  = document.createEvent('MouseEvents');
-
-    link.setAttribute('href', url);
-    link.setAttribute('target', '_blank');
-    link.setAttribute('download', 'data.csv');
-
-    evt.initMouseEvent('click', true, true, document.defaultView, 1, 0, 0, 0, 0,
-      false, false, false, false, 0, null);
-    link.dispatchEvent(evt);
+    var filename = "data.csv";
```
Comment (samuelcole) [inline src/components/MainView.js]: In ES6 we have `let` and `const`, which provide much more obvious block level scoping. Let's also prefer `const` where we can: https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75#.srb3gi4wz

File: src/components/MainView.js [hunk 2]
```diff
@@ -33,19 +33,20 @@ export default class MainView extends React.Component {
...
+    var csv = Papa.unparse(json);
+    var blob = new Blob([csv], {type: 'text/csv'});
+    if (window.navigator.msSaveOrOpenBlob)  // IE hack; see http://msdn.microsoft.com/en-us/library/ie/hh779016.aspx
```
Comment (samuelcole) [inline src/components/MainView.js]: just dumb style stuff: every `if` should have brackets on the same line as the `if`. so `if (window.navigator.msSaveOrOpenBlob) {` here and `} else {` on line 41.

i'll probably add eslint tonight to make sure our code style is consistent.

**Comment by samuelcole** [comment]
Just some style comments, but the logic looks good! 👍

**Comment by decause** [comment]
:+1:

File: src/components/MainView.js (hunk 2 above)
Comment (alexyaseen) [inline src/components/MainView.js]: I believe I already did add an eslint file. I have an eslint package in atom, but we should add a script to run it for those who don't.

**Comment by kahneraja** [comment]
on it! thanks for the feedback :)

File: src/components/MainView.js (hunk 2 above)
Comment (kahneraja) [inline src/components/MainView.js]: ah. i'm using sublime.

**Pull request merged**